tutorial/README.md

```diff
@@ -0,0 +1,221 @@
+# Earthmover Tutorial
+
+# Overview
+
+## What is this document?
+
+This document is a step by step tutorial introducing you to Earthmover as a tool - its components and how to run it. The target audience is a data engineer who has never used Earthmover before and who is going to be developing an EM assessment bundle.
```

Comment:
Is this just for internal EA DEs or is it also intended for external users?

Reply:
Good question. I wrote it with EA DE's in mind, but I suppose since this repo is public, a public audience should be considered.

Reply:
Alternatively, I had originally written this all in slite. If we don't feel ready to have something public, we could move it back to slite (very easy because it's all markdown)